spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -123,6 +123,17 @@ sealed class SpdxExpression {
      */
     abstract fun validate(strictness: Strictness)
 
+    /**
+     * Return all valid license choices for this SPDX expression, by converting it to the
+     * [disjunctive normal form][disjunctiveNormalForm] and collecting all disjunct expressions.
+     */
+    fun validChoices(): Set<SpdxExpression> = this.disjunctiveNormalForm().validChoicesDnf()
+
+    /**
+     * Internal implementation of [validChoices], assuming that this expression is already in disjunctive normal form.
+     */
+    protected open fun validChoicesDnf(): Set<SpdxExpression> = setOf(this)
```

Comment:
The function name, without reading the docs, could be misinterpreted to run `disjunctiveNormalForm()` itself. How about `validChoicesForDnf()`?

Reply:
Done.